packages/server-wallet/src/env.ts

```diff
@@ -0,0 +1,77 @@
+// FIXME: To remove @statechannels/hub's dependency on @statechannels/devtools,
+// this file is a carbon copy of the file that exports configureEnvVariables in devtools.
+
+// const {configureEnvVariables} = require('@statechannels/devtools');
```

Comment:
I'm going to use `configureEnvVariables` since we don't have the issue that required this hack

Reply:
80a2499